adapters/webhdfs-adapter/src/test/java/com/connexta/replication/adapters/webhdfs/WebHdfsNodeAdapterTest.java

```diff
@@ -523,4 +690,13 @@ public void testUpdateResourceNullResource() {
 
     assertThat(webHdfsNodeAdapter.updateResource(updateStorageRequest), is(false));
   }
+
+  private String readInputStreamToString(InputStream contentStream) throws IOException {
```

Comment:
Suggest using code below, both for using try-with-resources and to avoid manipulating content from the reader:
```
try (final Reader reader = new InputStreamReader(contentStream)) {
      return IOUtils.toString(reader);
    } catch (IOException e) {
      LOGGER.error("Failed to parse the HTTP content stream.", e);
      return null;
    }
```